ros_buildfarm/templates/status/blocked_releases_page.html.em

```diff
@@ -0,0 +1,123 @@
+        <script language="JavaScript">encoded_query_a_tag('label="UNRELEASED"', "Repositories that have not been released", "unreleased")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="BLOCKED"', "Repositories that are blocked from being released because of unreleased dependencies", "blocked")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED"', "Repositories that can be released", "releasable")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_BLOCKING"', "Repositories that can be released and are preventing others from being released", "releasable and blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('label="UNBLOCKED_UNBLOCKING"', "Repositories that can be released and are not preventing others from being released", "releasable and not blocking")</script>,
+        <script language="JavaScript">encoded_query_a_tag('id="metapackages"', "Repositories that are dependencies of the metapackages repository", "metapackages")</script>
+      </p>
+      <p id="search-count"></p>
+    </form>
+  </div>
+
+  <div class="top age">
+    <p></p>
+  </div>
+
+  <div class="table-div">
+    <table>
+      <thead>
+        <tr>
+          <!-- warning: if titles run onto two lines, first row of table data may be affected -->
+          <th class="sortable"><div>Repository</div></th>
+          <th class="sortable"><div>Version</div></th>
+          <th class="sortable">
+            <div title="Number of unreleased repositories that are directly blocking this one">
+              # blocking release
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Unreleased repositories that are directly blocking this one">
+              Blocking repos
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Maintainers of the unreleased repositories that are directly blocking this one">
+              Maintainers of blocks
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Number of unreleased repositories that are directly or indirectly blocking this one">
+              # recursively blocked
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Number of repositories that are being directly blocked by this one">
+              # directly blocked
+            </div>
+          </th>
+          <th class="sortable">
+            <div title="Repositories that are being directly blocked by this one">
+              Directly blocked repos
+            </div>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <!-- Originally sort the table by the repos with the most recursively blocked dependencies -->
+        <script type="text/javascript">window.sort=6; window.reverse=1;</script>
```

Comment:
I would suggest a secondary column to sort by (the name) since there can be rows with the same values and depending in the selected filter all rows might have no blocked deps.